app/main.py

```diff
@@ -1 +1,66 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = self.skills if self.skills else [
```

Comment:
self.skills is an empty list inherited from its parent. you just need to extend the list with the certain skill related to a particular class. 

Use += operator or extend() to extend a list instead of redefining it